src/app/shared/money-input/money-input.component.css

```diff
@@ -1,3 +1,11 @@
+.input-body {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  margin-top: 12px;
+}
+
 input {
   text-align: center;
   background: var(--text-color);
```

Comment:
What color is this? Rename it

Reply:
This color names is taken from the design in Figma and the project documentation.
Should i change it?